src/algebra/lie/of_associative.lean

```diff
@@ -66,34 +66,47 @@ lemma of_associative_ring_bracket (x y : A) : ⁅x, y⁆ = x*y - y*x := rfl
 
 end lie_ring
 
-namespace lie_algebra
+section lie_algebra
 
 variables {R : Type u} [comm_ring R] [algebra R A]
 
 /-- An associative algebra gives rise to a Lie algebra by taking the bracket to be the ring
 commutator. -/
 @[priority 100]
-instance of_associative_algebra : lie_algebra R A :=
+instance lie_algebra.of_associative_algebra : lie_algebra R A :=
```

Comment:
Out of scope for this PR, but these might make sense phrased in the other direction, `algebra.to_lie_algebra` to match `semiring.to_module` etc

Reply:
I agree this naming deserves fixing but I would rather leave it as future work.